server/routes/api/documents/documents.ts

```diff
@@ -435,6 +436,52 @@ router.post(
   }
 );
 
+router.post(
+  "documents.users",
```

Comment:
Should we paginate this yet? :thinking:

Reply:
Yes, unfortunately we have 1000+ member collections to think of